File: Ticket.java
public class Ticket extends Servicios{
    private int codigo;
    private int tipoTicket;
    private String descripcion;

    // Getter
    public int getCodigo() {
        return codigo;
    }

    public int getTipoTicket() {
        return tipoTicket;
    }

    public String getDescripcion() {
        return descripcion;
    }

    // Setter

    public void setCodigo(int newCodigo) {
        this.codigo = newCodigo;
    }

    public void setTipoTicket(int newtipoTicket) {
        this.tipoTicket = newtipoTicket;
    }

    public void setDescripcion(String newdescripcion) {
        this.descripcion = newdescripcion;
    }
}

File: src/js/ticket.js
import { Servicios } from './servicios.js';

export class Ticket extends Servicios {
    codigo;
    tipoTicket;
    descripcion;
    constructor(codigo, tipoTicket, descripcion) {
        this.codigo = codigo;
        this.tipoTicket = tipoTicket;
        this.descripcion = descripcion;
    }

    get getCodigo() {
        return codigo;
    }

    set setCodigo(newCodigo) {
        this.codigo = newCodigo;
    }

    get getTipoTicket() {
        return tipoTicket;
    }

    set setTipoTicket(newTipoTicket) {
        this.tipoTicket = newTipoTicket;
    }

    get getDescripcion() {
        return descripcion;
    }

    set setDescripcion(newDescripcion) {
        this.descripcion = newDescripcion;
    }
}